tests/integration/returners/test_librato_return.py

```diff
@@ -2,62 +2,65 @@
 """
 Tests for the librato returner
 """
-# Import Python libs
 from __future__ import absolute_import, print_function, unicode_literals
 
 import logging
 
-# Import salt libs
+import pytest
 from salt.returners import librato_return
-
-# Import Salt Testing libs
 from tests.support.case import ShellCase
 
 log = logging.getLogger(__name__)
 
-# JOBS DIR and FILES
-MOCK_RET_OBJ = {
-    "fun_args": [],
-    "return": {
-        "test-return-state": {
-            "comment": "insertcommenthere",
-            "name": "test-state-1",
-            "start_time": "01: 19: 51.105566",
-            "result": True,
-            "duration": 3.645,
-            "__run_num__": 193,
-            "changes": {},
-            "__id__": "test-return-state",
-        },
-        "test-return-state2": {
-            "comment": "insertcommenthere",
-            "name": "test-state-2",
-            "start_time": "01: 19: 51.105566",
-            "result": False,
-            "duration": 3.645,
-            "__run_num__": 194,
-            "changes": {},
-            "__id__": "test-return-state",
-        },
-    },
-    "retcode": 2,
-    "success": True,
-    "fun": "state.highstate",
-    "id": "Librato-Test",
-    "out": "highstate",
-}
-
 
+@pytest.mark.windows_whitelisted
 class libratoTest(ShellCase):
     """
     Test the librato returner
     """
 
+    @classmethod
+    def setUpClass(cls):
+        cls.mock_ret_obj = {
+            "fun_args": [],
+            "return": {
+                "test-return-state": {
+                    "comment": "insertcommenthere",
+                    "name": "test-state-1",
+                    "start_time": "01: 19: 51.105566",
+                    "result": True,
+                    "duration": 3.645,
+                    "__run_num__": 193,
+                    "changes": {},
+                    "__id__": "test-return-state",
+                },
+                "test-return-state2": {
+                    "comment": "insertcommenthere",
+                    "name": "test-state-2",
+                    "start_time": "01: 19: 51.105566",
+                    "result": False,
+                    "duration": 3.645,
+                    "__run_num__": 194,
+                    "changes": {},
+                    "__id__": "test-return-state",
+                },
+            },
+            "retcode": 2,
+            "success": True,
+            "fun": "state.highstate",
+            "id": "Librato-Test",
+            "out": "highstate",
+        }
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.mock_ret_obj = None
+
     def test_count_runtimes(self):
         """
         Test the calculations
         """
-        results = librato_return._calculate_runtimes(MOCK_RET_OBJ["return"])
+        results = librato_return._calculate_runtimes(self.mock_ret_obj["return"])
```

Comment:
Does this `mock_ret_obj` change need to be in this PR? Also, since pytest, shouldn't this just be a pytest fixture?

Reply:
> Does this mock_ret_obj change need to be in this PR? 

Module level globals, specially dictionaries, are evil. If you really want me to take this into the "BIG PR", I'll take it out.

> Also, since pytest, shouldn't this just be a pytest fixture?
When we have a full pytest only test suite, sure. I'm not going to migrate every single test I touch, at least just yet :grinning:

Reply:
I think if you pull these two changes into a smaller PR - or even one that's a bit larger, that's doing a bunch of this kind of cleanup - that would be better. If this one is *just* adding windows whitelisting markers, that's :+1: 

Then we can have other(s) that are doing other assorted cleanup.

Reply:
backed out